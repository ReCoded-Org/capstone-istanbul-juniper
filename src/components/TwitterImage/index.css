.twitterContent__col__image {
  min-width: 80%;
  max-width: 92%;
  margin-top: 5%;
  margin-bottom: 5%;
  transition: all 0.5s;
  cursor: pointer;
  max-height: 530px;
}

.twitterContent__col__image:hover {
  transform: scale(1.1);
  transition: all 0.5s;
}
