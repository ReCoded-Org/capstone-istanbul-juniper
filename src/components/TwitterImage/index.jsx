import React from "react";
import "./index.css";

const TwitterImage = ({ src, href, alt }) => {
  return (
    <a href={href} target="_blank" rel="noopener noreferrer">
      <img src={src} alt={alt} className="twitterContent__col__image" />
    </a>
  );
};

export default TwitterImage;
