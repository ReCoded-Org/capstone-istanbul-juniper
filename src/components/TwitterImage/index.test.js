import React from "react";
import renderer from "react-test-renderer";
import TwitterImage from ".";

it("TwitterImage renders correctly", () => {
  const tree = renderer.create(<TwitterImage />).toJSON();
  expect(tree).toMatchSnapshot();
});
