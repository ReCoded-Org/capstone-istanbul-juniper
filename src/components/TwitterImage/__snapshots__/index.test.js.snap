// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TwitterImage renders correctly 1`] = `
<a
  rel="noopener noreferrer"
  target="_blank"
>
  <img
    className="twitterContent__col__image"
  />
</a>
`;
